lib/JumpProblemLibrary/Project.toml

```
@@ -1,6 +1,6 @@
 name = "JumpProblemLibrary"
 uuid = "faf0f6d7-8cee-47cb-b27c-1eb80cef534e"
-version = "0.1.4"
+version = "0.2"
```

```suggestion
version = "1.0.0"
```

let's not keep everything 0-based forever 😅